app/auth/auth_service.py

```
@@ -39,11 +46,19 @@ def verify_password(plain_pw: str, hashed_pw: str) -> bool:
 
 def create_access_token(user: UserBase) -> Token:
     expire = datetime.now(timezone.utc) + timedelta(minutes=ACCESS_TOKEN_EXPIRE_MIN)
-    to_encode = {"sub": user.username, "exp": expire}
+    to_encode = {"sub": user.username, "exp": expire, "type": "access"}
     encoded_jwt = jwt.encode(to_encode, SECRET_KEY, algorithm=ALGORITHM)
     return Token(access_token=encoded_jwt, token_type="bearer")
```

Ruff (0.15.13)

[error] Possible hardcoded password assigned to argument: "token_type"

(S106)